package.json

```diff
@@ -45,7 +45,8 @@
     "widget"
   ],
   "dependencies": {
-    "react": "0.13.x",
+    "react": "0.14.x",
```

Comment:
If we actually don't have to change anything about the plugin to make it compatible with 0.14, can we use a more flexible version range (cf. https://docs.npmjs.com/files/package.json#dependencies)? Couldn't the user have *either* 0.13.x *or* 0.14.x?